src/renderers/webgl/WebGLPrograms.js

```diff
@@ -110,7 +110,16 @@ function WebGLPrograms( renderer, extensions, capabilities ) {
 
 	}
 
-	this.getParameters = function ( material, lights, shadows, fog, nClipPlanes, nClipIntersection, object ) {
+	this.getParameters = function ( material, lights, shadows, scene, nClipPlanes, nClipIntersection, object ) {
+
+		var fog = scene.fog;
+		var envMap = material.envMap;
+
+		if ( material.isMeshStandardMaterial ) {
+
+			if ( scene.background && scene.background.isCubeTextureBackground ) envMap = scene.background.texture;
```

Comment:
So this means setting a background will clobber any envMaps set on the materials themselves? Should this be reversed so that if I really want a particular envMap on a material, I can specify it and have that take precedence, but the normal path is to leave envMap unspecified and just use scene.background?

Reply:
Good point, I can change the logic so it does:

```js
envMap = material.envMap || scene.background.texture;
```